modules/tls-app-v2/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "produce_topics" {
+  type        = list(string)
+  description = "The topics that the app produces to"
+  default     = []
+}
+
+variable "consume_topics" {
+  type        = list(string)
+  description = "The topics that the app consumes."
+  default     = []
+}
+
+variable "consume_groups" {
+  type        = list(string)
+  description = "The Kafka consumer groups that the app uses to consume."
+  default     = []
+}
+
+variable "cert_common_name" {
+  type        = string
+  description = "The principal or user/group for which the ACL is being created."
+}
+
+# ACL
+variable "acl_host" {
+  type        = string
+  description = "The host from which the principal is allowed to perform operations. Set to '*' to allow any host."
+  default     = "*"
+}
+
+# QUOTA
+variable "consumer_byte_rate" {
```

Comment:
consumer_byte_rate and producer_byte_rate descriptions need updating

Reply:
Done